urlpattern/resources/urlpatterntestdata.json

```diff
@@ -1874,7 +1875,17 @@
   {
     "pattern": [ "https://{sub.}?example{.com/}foo" ],
     "inputs": [ "https://example.com/foo" ],
-    "expected_obj": "error"
```

Comment:
This was just invalid right? Is there any evidence to why this was introduced in the first place?